rules/windows/discovery_high_number_ad_properties.toml

```diff
@@ -0,0 +1,67 @@
+[metadata]
+creation_date = "2023/01/29"
+integration = ["windows", "system"]
```

Comment:
Is this telemetry visible with Elastic Defend? If so, we can add the integration.

Reply:
This one is AD related.